src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/ModelAttributes.cs

```diff
@@ -32,17 +33,56 @@ public static IEnumerable<object> GetAttributesForParameter(ParameterInfo parame
         /// <summary>
         /// Gets the attributes for the given <paramref name="property"/>.
         /// </summary>
-        /// <param name="property">A <see cref="ParameterInfo"/> for which attributes need to be resolved.
+        /// <param name="type">A <see cref="Type"/> on which the property exists.
```

Comment:
- need `</param>` at the end.  invalid XML as-is
- nit: `The <see cref="Type"/> containing <paramref name="property"/>.` or perhaps `The <see cref="Type"/> in which caller found <paramref name="property"/>.`